modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -2,6 +2,13 @@
 //
 // * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
 
+ifeval::["{context}" == "installing-aws-user-infra"]
```

Comment:
Because just because a file's originally created with one use in mind doesn't mean it won't be picked up somewhere else, and with my available information, that step's only necessary for AWS.